Formula/pass-otp.rb

```diff
@@ -0,0 +1,39 @@
+class PassOtp < Formula
+  desc "The Pass extension for managing one-time-password (OTP) tokens"
+  homepage "https://github.com/tadfisher/pass-otp#readme"
+  url "https://github.com/tadfisher/pass-otp/releases/download/v1.1.1/pass-otp-1.1.1.tar.gz"
+  sha256 "edb3142ab81d70af4e6d1c7f13abebd7c349be474a3f9293d9648ee91b75b458"
+
+  depends_on "oath-toolkit"
+  depends_on "pass"
```

Comment:
Thanks for building this! According to the [pass-otp README](https://github.com/tadfisher/pass-otp#requirements) this should also depend on `qrencode` which is useful for some mobile apps.

Reply:
Thanks Mike! They do mention that requirement in README, but I have just check source code pass-otp actually never calls `qrencode`. Instead, it calls a function wrapping showing QR image, that is built into `pass`. You can heve a look [here](https://github.com/tadfisher/pass-otp/blob/7d7a9afc48562ac292e6847ebe834ed673e83ebd/otp.bash#L406).

The `pass-otp` already depends on `pass`, which in turns depends on `qrencode` to implement the aforementioned function. I think it is better to leave it the way it is, so we are not locking to a dependency, that they may decide to change in feature releases of `pass`. Don't you agree?

Reply:
Oh cool, I didn't look at the code. I agree, since they're calling through to `pass` and `qrencode` is not a direct dependency of `pass-otp` then it shouldn't be declared here. Seems like a bug in the `pass-otp` README.

+1 from me for merging this